src/components/Editable.js

```diff
@@ -62,6 +62,33 @@ const EMPTY_THOUGHT_TIMEOUT = 5 * 1000
 // eslint-disable-next-line jsdoc/require-jsdoc
 const stopPropagation = e => e.stopPropagation()
 
+/** Add position:absolute to toolbar elements in order to fix Safari position:fixed browser behavior when keyboard is up. */
+const makeToolbarPositionFixed = () => {
+  const hamburgerMenu = document.getElementsByClassName('hamburger-menu')[0]
+  const toolbar = document.getElementsByClassName('toolbar-container')[0]
+  Array.from([hamburgerMenu, toolbar]).forEach(el => {
+    el.style.position = 'absolute'
+    el.style.top = `${window.scrollY}px`
+  })
+}
+/** Reset position:absolute of toolbar elements. */
+const resetToolbarPosition = () => {
+  const hamburgerMenu = document.getElementsByClassName('hamburger-menu')[0]
+  const toolbar = document.getElementsByClassName('toolbar-container')[0]
+  Array.from([hamburgerMenu, toolbar]).forEach(el => {
+    el.style.position = 'fixed'
+    el.style.top = ''
+  })
+}
+/** Update position of toolbar elements while scrolling in order to show them always on top. */
+const updateToolbarPositionOnScroll = () => {
+  const hamburgerMenu = document.getElementsByClassName('hamburger-menu')[0]
+  const toolbar = document.getElementsByClassName('toolbar-container')[0]
+  Array.from([hamburgerMenu, toolbar]).forEach(el => {
+    el.style.top = `${window.scrollY}px`
+  })
+}
+
```

Comment:
Could this be done in the HamburgerMenu and Toolbar components via `state` and the appropriate reducers? I would prefer a more idiomatic react approach than direct DOM manipulation.

Reply:
Yes, i wrote this for testing, i can reorganize this code in any way.

Reply:
Sounds good. Once we get the functionality perfected than we can turn our attention to architecture.